recipes/r-scevan/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: r-scevan
+  version: 1.0.1
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/v1.0.1.tar.gz  # Replace with the GitHub URL of the release
+  sha256: 7558d31282c36418db3960b3edb9cb9815db14b0a48d8882287e11f752dfeab1       # Generate the checksum using 'sha256sum' command
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("SCEVAN", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "$R -e \"install.packages(c('Rcpp', 'remotes', 'devtools'), lib='$PREFIX/lib/R/library', repos='https://cran.rstudio.com/')\""
+
+requirements:
+  host:
+    - r
+    - r-base
+    - r-remotes
+    - r-devtools
+  run:
+    - r
+    - r-base
+    - r-devtools
+    - r-paralleldist
+    - r-pheatmap
+    - r-forcats
+    - r-dplyr
+    - bioconductor-fgsea
+# - r-yaGST - manually with devtools
+    - r-cluster
+    - r-ggplot2
+    - r-Rtsne
+    - bioconductor-scran
+    - r-ape
+    - bioconductor-ggtree
+    - r-tidytree
+    - r-ggrepel
+    - r-optparse
+
+test:
+  commands:
+    - $R -e "library(devtools)"
+    - $R -e "devtools::install_github('miccec/yaGST')"
+    - $R -e "devtools::install_github('AntonioDeFalco/SCEVAN', ref = 'v1.0.1')"
+    - $R -e "library(SCEVAN)"
+
+about:
+  home: https://github.com/AntonioDeFalco/SCEVAN/
+  license: GNU GPLV3
```

Comment:
please use SPDX identifiers

Reply:
updated license to 'GPL-3.0-only' identifier